.circleci/config.yml

```diff
@@ -540,6 +583,23 @@ jobs:
           enable-dashboard: << parameters.enable-dashboard >>
           suite: << parameters.suite >>
 
+  #Maybe have a move this to a image folder
+  docker-deploy:
+    docker:
+      - image: cimg/node:16.16.0
+    steps:
+      - set-dev-env-vars
+      - docker-deploy-image
+
+  ecr-deploy:
```

Comment:
I don't see this job being used anywhere. If its not being used, it should be removed

Reply:
Removed the extra job.